vbench/src/vbench/vbench/worker.cpp

```diff
@@ -5,6 +5,8 @@
 
 namespace vbench {
 
+VESPA_THREAD_STACK_TAG(vbench_worker_executor);
```

Comment:
Consider _thread instead of _executor as this is used for vespalib::Thread.

Reply:
Fixed